cli/cli/installer_std_controller.go

```diff
@@ -15,11 +15,15 @@ import (
 func installSelfCert(ctx apitypes.Context, config gofig.Config) {
 	certPath := config.GetString(apitypes.ConfigTLSCertFile)
 	keyPath := config.GetString(apitypes.ConfigTLSKeyFile)
-	host := "127.0.0.1"
+
+	host, err := os.Hostname()
+	if err != nil {
+		ctx.Fatalf("failed to get hostname for cert")
+	}
 
 	fmt.Println("Generating server self-signed certificate...")
 	if err := util.CreateSelfCert(ctx, certPath, keyPath, host); err != nil {
-		log.Fatalf("cert generation failed: %v\n", err)
+		ctx.Fatalf("cert generation failed: %v\n", err)
```

Comment:
Hi @vladimirvivien,

Could you possibly use the logger's `WithError` function?

```go
ctx.WithError(err).Fatal("cert generation failed")
```